content/reference-transaction-format.md

```diff
@@ -11,7 +11,7 @@ A _Transaction_ is the only way to modify the Ripple Ledger. Transactions are on
 
```

Comment:
We should change "Ripple Ledger" in [line 3](https://github.com/ripple/ripple-dev-portal/pull/263/files#diff-d99a4557e170059e331d3ce7b5ee7338R3) to "XRP Ledger".